storage/innobase/handler/handler0alter.cc

```diff
@@ -4565,15 +4569,6 @@ static MY_ATTRIBUTE((warn_unused_result)) bool prepare_inplace_alter_table_dict(
 
     const char *encrypt;
     encrypt = ha_alter_info->create_info->encrypt_type.str;
-    /* If encryption option is specified, then it must be
-    innodb-file-per-table tablespace. Otherwise case would
-    have already been blocked at
-    create_option_tablespace_is_valid(). */
-    if (encrypt) {
```

Comment:
why is this block removed?

Reply:
upstream change - It is no longer is valid that only innodb-file-per-table tablespaces can be encrypted.

Reply:
if upstream, then OK